jobs/periodic/sync-to-webflow/webflow-adapter.ts

```diff
@@ -0,0 +1,81 @@
+import { getLogger } from '../../utils/logging';
+import { join } from 'path';
+
+const logger = getLogger();
```

Comment:
You could pass a topic here, i.e getLogger("Webflow Sync")

Reply:
Hm, not sure if we ever want to replace log4js, and even then we already have hundreds of places where we directly import `getLogger` from it, so I guess it's just fine to also do so here.